packages/frontend/.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,23 @@
+404.html,1591747591833,b7bab6b83fa074653ff28c8d2a64135d3434575f70a12ab3d3ba8080461b9537
+flaky-dashboard/favicon.ico,1591746399596,5ba04cf43d3c8c026bd5edd43046db96acb28b5686d99ea7d7e148feabfffc8f
+flaky-dashboard/index.html,1591746399596,9c65f8872da56c1dba66e92931fd6ee05b5dfcdbbc3f63b0e922b99d70ea4717
+flaky-dashboard/main-es2015.js,1591746364694,67f5510d8c7cf7199bc13a3bac7af11dd6e5c5c0caccf5d327208c7840c00d2e
+flaky-dashboard/main-es2015.js.map,1591746364694,ca51fc7528135e48b38fc8ee9f25c93c89cc20bfb6b487281edd4c5762c63aa9
+flaky-dashboard/main-es5.js,1591746364694,dc2e89dbe2cc314a51ff11054ffd6eaf1decb1680283df2504a4e55d314f1d1a
+flaky-dashboard/main-es5.js.map,1591746364694,0d0662076088644344a2eeeb71bc8166d72570a40baf0ca868b05a0ba0f53df3
+flaky-dashboard/polyfills-es2015.js,1591746364694,e0ca47e773d9520e5ed5e267c3e5f63c5299d0fa3080fcd8e4f3acf287d2db97
```

Comment:
we should probably add the `.firebase` folder to `.gitignore` in the top level, I think you can just add the line:

```
.firebase
```

and then run `git rm --force packages/frontend/.firebase`.